services/brig/test/integration/Federation/End2end.hs

```diff
@@ -210,12 +210,12 @@ testClaimMultiPrekeyBundleSuccess brig1 brig2 = do
       const 200 === statusCode
       const (Just ucm) === responseJsonMaybe
 
-testAddRemoteUsersToLocalConv :: Brig -> Galley -> Brig -> Http ()
-testAddRemoteUsersToLocalConv brig1 galley1 brig2 = do
+testAddRemoteUsersToLocalConv :: Brig -> Galley -> Brig -> Galley -> Http ()
+testAddRemoteUsersToLocalConv brig1 galley1 brig2 galley2 = do
   alice <- randomUser brig1
   bob <- randomUser brig2
 
-  let conv = NewConvUnmanaged $ NewConv [] [] (Just "gossip") mempty Nothing Nothing Nothing Nothing roleNameWireAdmin
+  let newConv = NewConvUnmanaged $ NewConv [] [] (Just "gossip") mempty Nothing Nothing Nothing Nothing roleNameWireAdmin
   convId <-
     cnvId . responseJsonUnsafe
       <$> post
```

Comment:
I was about to propose to use `postConvWithRoleQualified` instead, but then I am not sure how it would fit the `Http` monad and even more importantly, how to make sure `galley1` is used, and not `galley2`. What's perhaps even more important is that this function is located in Galley integration tests.

Reply:
At this point we probably want to duplicate some of these helper functions; but I agree it would be nice to consolidate things more. If/when we merge galley+brig+spar, that will become a non-issue.